src/components/Home/Libraries.js

```diff
@@ -29,6 +29,11 @@ const librariesData = {
                 url: '/docs/libraries/vue',
                 icon: <StaticImage src="../../../contents/images/docs/integrate/frameworks/vue.svg" alt="Vue" />,
             },
+            {
```

Comment:
This list isn't exhaustive and it is a mix of a single SDK (JS) and many frameworks, so makes sense to point out to More->frameworks